### ISSUE TITLE
Ask some questions about SpotTune

### ISSUE THREAD

**Issue by JasonCLEI** [gyhui14/spottune#1]
Guo博，您好，在跑您release在Github上的代码时发现代码实现和论文阐述的思想并不完全一致，深感疑惑。您在论文中说的是Original Blocks完全freeze住，参数不更新，Parallel Blocks进行参数更新，blocks的选择取决于Policy network产生的actions。但在代码实现中您把Original Blocks中的Conv2d和Parallel Blocks中的Conv2d的weight指向了同一地址空间，导致后面的requires_grad=False并没有起作用，Original Blocks中的Conv2d和Parallel Blocks中的Conv2d的weight一直在更新并且始终一致，但Original Blocks中的BatchNorm2d和Parallel Blocks中的BatchNorm2d的参数又都是单独训练的，且都没有freeze，相当于您Original Blocks和Parallel Blocks中的Conv2d是share并共同更新的，BatchNorm2d又是分开单独训练的，没有哪个部分是freeze住的，这和您论文里提出的思想并不是一致的。希望您能给我解答一下，万分感谢！

**Comment by gyhui14** [comment]
Thanks for pointing out. There is a  typo occurred when I cleaned up the code. The code is updated and should produce the score reported.
